### DISCUSSION TITLE
MS Store updates not shown ?

### DISCUSSION THREAD

**Discussion by nolme** [Devolutions/UniGetUI#2269] [Q&A] [answered]
Hello again.
Is it possible to have MS Store updates listed too ?
![WinGetUI](https://github.com/marticliment/WingetUI/assets/4461224/0789f893-ed4e-4002-bb16-59d475d24d5a)
As you can see, 2 updates are missing.
I think this is because source is only WinGet so how can I add MS Store ?
Thanks,
Vincent

**Comment by marticliment** [comment]
Hello,

WinGet does not list Microsoft Store updates

**Reply by nolme** [reply]
But when I type :
`winget source list`
I get : 
```
Nom     Argument
-----------------------------------------------------
msstore https://storeedgefd.dsx.mp.microsoft.com/v9.0
winget  https://cdn.winget.microsoft.com/cache
```

**Reply by marticliment** [reply, marked as answer]
Yes, but the Microsoft Store source does not provide updates for regular microsoft store packages. Updates for the microsoft store packages will be shown only for the EXE/MSI microsoft store apps
